### ISSUE TITLE
Bug reading config file?

### ISSUE THREAD

**Issue by howesteve** [dosco/super-graph#64]
Hello,

On the most recent version, there seems to be a bug finding the config file (function newViper()) if a file extension is provided.

``` go
conf, err := core.ReadInConfig("supergraph") // => finds the config file
conf, err := core.ReadInConfig("supergraph.yaml") // => returns an error, config file not find.
```

The following seems to fix it.
``` go
func newViper(configPath, configFile string) *viper.Viper {
	vi := viper.New()

	vi.SetEnvPrefix("SG")
	vi.SetEnvKeyReplacer(strings.NewReplacer(".", "_"))
	vi.AutomaticEnv()

	if filepath.Ext(configFile) != "" {
		vi.SetConfigFile(configFile)
	} else {
		vi.SetConfigName(configFile)
		vi.AddConfigPath(configPath)
		vi.AddConfigPath("./config")
	}

	return vi
}
```

**Comment by dosco** [comment]
Good catch this is a feature of the config library `viper` that I failed to handle. Do you want to submit a PR or should I make the fix either works.

**Comment by howesteve** [comment]
Since it's a couple lines, it's probably easier if you change. Let me know if you prefer a PR. 

Thanks.

**Comment by dosco** [comment]
I'll get to it later today. I was just asking that way credit does where it's due :) Thanks again great catch it's these little corner cases that really help polish up a project.

**Comment by dosco** [comment]
I committed a fix 3bf9f02a9f4b0bc208ddffa7ad16d5e5a91cfe81

**Comment by howesteve** [comment]
Thanks!
